examples/basic/main.tf

```diff
@@ -16,7 +21,16 @@ module "one" {
 module "for_each" {
   for_each  = toset(["a", "b", "c"])
   source    = "../.."
-  providers = { aws = aws, aws.kms = aws }
+  providers = { aws = aws, aws.kms = aws, aws.dns_query_logging = aws.dns_query_logging }
   name      = "${each.key}.example.org"
   tags      = {}
 }
+
+// Test single zone with DNS query logging enabled
```

Comment:
I'm guessing this should not be in here?

Reply:
Do you mean the comment or the module?

Reply:
This is in examples :D Overlooked that, sorry.